setup.py

```diff
@@ -16,7 +16,7 @@
     author='Michael Rooney',
     author_email='mrooney.mintapi@rowk.com',
     url='https://github.com/mrooney/mintapi',
-    install_requires=['future', 'mock', 'requests', 'selenium-requests', 'xmltodict', 'pandas>=1.0', 'selenium'],
+    install_requires=['future', 'mock', 'requests', 'selenium-requests', 'xmltodict', 'pandas>=1.0', 'selenium', 'oathtool'],
```

Comment:
Since you only import it if that argument is passed, technically it doesn't require it, but I guess why not?

Reply:
You're right, I just didn't think it through after making the other change.

Reply:
Hey, @mrooney I think you wanted to remove this before merge?

Reply:
> Hey, @mrooney I think you wanted to remove this before merge?

It's okay, I think it is fine there!